src/components/pages/SignIn/email/components/PasswordTextInput.tsx

```
@@ -0,0 +1,45 @@
+import React, { FC } from 'react';
+import { useFormContext } from 'react-hook-form';
+
+import { ErrorMessage } from '@hookform/error-message';
+import { TextInput } from 'src/components/blocks';
+
+interface PasswordTextInputProps {}
+
+export const PasswordTextInput: FC<PasswordTextInputProps> = ({}) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)